docs/reference/remap/functions/truncate.cue

```diff
@@ -20,18 +20,21 @@ remap: functions: truncate: {
 			type: ["integer", "float"]
 		},
 		{
-			name:        "ellipsis"
-			description: "If true, an ellipsis (...) is appended should the string be truncated."
-			required:    true
+			name: "ellipsis"
+			description: """
+				An ellipsis (`...`) is appended if this is set to `true` _and_ the `value` string ends up being
+				truncated because it's exceeded the `limit`.
+				"""
+			required: true
 			type: ["boolean"]
 		},
 	]
 	internal_failure_reasons: []
 	return: {
 		types: ["string"]
 		rules: [
-			"If `limit` is larger than the length of the string, the string is returned unchanged.",
-			"If `ellipsis` is `true`, then an ellipsis (...) will be appended to the string (beyond the specified limit).",
+			"The string is returned unchanged its length is less than `limit`.",
```

Comment:
```suggestion
			"The string is returned unchanged if its length is less than `limit`.",
```

Reply:
I merged this yesterday to get it through for the release but I'll follow up